tests/e2e/config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "src_folders": ["./tests/e2e/spec"],
+  "output_folder": "./tests/e2e/logs",
+  "custom_commands_path" : "node_modules/nightwatch-custom-commands-assertions/js/commands",
+  "custom_assertions_path" : "node_modules/nightwatch-custom-commands-assertions/js/assertions",
+  "page_objects_path": "",
+  "selenium": {
+    "start_process": true,
+    "server_path": "tests/e2e/bin/selenium.jar",
+    "log_path": "./tests/e2e/logs",
+    "host": "127.0.0.1",
+    "port": 4444,
+    "cli_args": {
+      "webdriver.chrome.driver": "tests/e2e/bin/chromedriver"
+    }
+  },
+  "test_settings": {
+    "default": {
+      "launch_url": "",
+      "silent": true,
+      "desiredCapabilities": {
+        "browserName": "chrome",
```

Comment:
Could we integrate with browser stack? That would help us write test cases specific to IE and test it